### ISSUE TITLE
[MNT] Release the package on PyPi

### ISSUE THREAD

**Issue by MatthewMiddlehurst** [aeon-toolkit/aeon-neuro#55]
### Describe the issue

The package is not currently uploaded to PyPi. While theres not much content currently, it is good to reserve the name and make sure the pipeline works for when there is.

### Suggest a potential alternative/fix

Publish the package to PyPI with a `0.0.1` release and make sure the release workflow works.

### Additional context

See:
- The `release.yml` workflow (https://github.com/aeon-toolkit/aeon-neuro/blob/main/.github/workflows/release.yml)
- The `aeon` PyPi (https://pypi.org/project/aeon/)
- The `aeon` GitHub releases (https://github.com/aeon-toolkit/aeon/releases)

**Comment by griegner** [comment]
@MatthewMiddlehurst 
To clarify the version for the first release, should it be `0.0.1` as listed in this issue or `0.1.0` as written in the `pyproject.toml`?
https://github.com/aeon-toolkit/aeon-neuro/blob/5987568966593e40179713f388282cb8a6fb9d41/pyproject.toml#L7

**Comment by MatthewMiddlehurst** [comment]
I don't think it really matters at this point, but I would go with `0.0.1` since we have very little content and its mostly just to get it on there rather than share anything.

**Comment by MatthewMiddlehurst** [comment]
@mloning may have a different opinion, it may be better to do something different altogether 🙂

**Comment by mloning** [comment]
No sounds good to me!
